apps/hes/fsw/mission_inc/hes_perfids.h

```diff
@@ -39,7 +39,8 @@ extern "C" {
 #endif
 
 /** \brief Performance ID for the TO main task */
-#define HES_MAIN_TASK_PERF_ID            (124)
```

Comment:
In the configuration header files in the "apps/<app_name>/fsw/<mission_inc/platform_inc>/" directories, you need to include the actual macro here, not our "#include "perf_ids.h".  The "perf_ids.h" technique is something we use for most of our Airliner, but is not universally used for all builds.  The "reference" build ("make reference"), builds using this file, not from the files in config directories.  They should pull the macros directly from these files because we use these files to generate the doxygen documentation.  These files should be fully doxy'd up because we use the "reference" build to generate mission/vehicle independent documentation.  The actual values contained within the macro are irrelevant.  We don't actually run the reference build.  You can give them any value you want.  Doesn't matter.